backend/src/test/java/com/woowacourse/levellog/presentation/FeedbackControllerTest.java

```diff
@@ -366,4 +367,88 @@ private ResultActions requestFindAllFeedback(final Long levellogId) throws Excep
             return requestGet("/api/levellogs/" + levellogId + "/feedbacks");
         }
     }
+
+    @Nested
+    @DisplayName("findById 메서드는")
+    class FindById {
+
+        @Test
+        @DisplayName("존재하지 않는 레벨로그 정보로 피드백 목록 조회를 요청하면 예외가 발생한다.")
+        void findById_notFoundLevellog_exceptionThrown() throws Exception {
+            // given
+            final Long memberId = 1L;
+            final Long feedbackId = 1L;
+            final Long levellogId = 200000L;
+
+            final String message = "레벨로그가 존재하지 않습니다.";
+            given(feedbackService.findById(levellogId, feedbackId, memberId))
+                    .willThrow(new LevellogNotFoundException(message));
+
+            // when
+            final ResultActions perform = requestFindByIdFeedback(levellogId, feedbackId);
+
+            // then
+            perform.andExpectAll(
+                    status().isNotFound(),
+                    jsonPath("message").value(message)
+            );
+
+            // docs
+            perform.andDo(document("feedback/find-by-id/exception/levellog-not-found"));
+        }
+
+        @Test
+        @DisplayName("속하지 않은 팀의 피드백 조회를 요청하면 예외가 발생한다.")
+        void findById_notMyTeam_exception() throws Exception {
+            // given
+            final Long memberId = 1L;
+            final Long feedbackId = 1L;
+            final Long levellogId = 1L;
+
+            final String message = "권한이 없습니다.";
+            given(feedbackService.findById(levellogId, feedbackId, memberId))
+                    .willThrow(new UnauthorizedException(message));
+
+            // when
+            final ResultActions perform = requestFindByIdFeedback(levellogId, feedbackId);
+
+            // then
+            perform.andExpectAll(
+                    status().isUnauthorized(),
+                    jsonPath("message").value(message)
+            );
+
+            // docs
+            perform.andDo(document("feedback/find-by-id/exception/not-my-team"));
+        }
+
+        @Test
+        @DisplayName("잘못된 레벨로그의 피드백 조회를 요청하면 예외가 발생한다.")
+        void findById_levellogWrongId_exception() throws Exception {
+            // given
+            final Long memberId = 1L;
+            final Long feedbackId = 1L;
+            final Long levellogId = 1L;
+
+            final String message = "입력한 levellogId와 피드백의 levellogId가 다릅니다.";
+            given(feedbackService.findById(levellogId, feedbackId, memberId))
+                    .willThrow(new InvalidFieldException(message));
+
+            // when
+            final ResultActions perform = requestFindByIdFeedback(levellogId, feedbackId);
+
+            // then
+            perform.andExpectAll(
+                    status().isBadRequest(),
+                    jsonPath("message").value(message)
+            );
+
+            // docs
+            perform.andDo(document("feedback/find-by-id/exception/levellog-wrong-id"));
```

Comment:
```suggestion
            perform.andDo(document(BASE_SNIPPET_PATH+"levellog-wrong-id"));
```
다음과 같이 Nested 내부에서 상수로 처리해주세요.